modules/tests/kubecf-test.sh

```diff
@@ -96,10 +96,23 @@ kubectl logs -f "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container "$co
 
 # Wait for the container to terminate and then exit the script with the container's exit code.
 jsonpath='{.status.containerStatuses[?(@.name == "'"$container_name"'")].state.terminated.exitCode}'
-while true; do
-  exit_code="$(kubectl get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
-  if [[ -n "${exit_code}" ]]; then
-      exit "${exit_code}"
-  fi
-  sleep 1
+exit_code="$(kubectl get "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --output "jsonpath=${jsonpath}")"
+while [[ -z "${exit_code}" ]]; do
```

Comment:
Wouldn't make sense to re-use the same approach used in SCF cats tests? https://github.com/SUSE/catapult/blob/master/modules/tests/kubecats.sh#L37

Reply:
doh, we didn't realize. thanks!

Reply:
`wait_container_attached()` is not useful as we need to save the exit code of the container. Going with Mark's simplifying suggestion.